src/pages/documentation/adobe-analytics/tabs/api-reference.md

```diff
@@ -384,24 +350,34 @@ Analytics.getVisitorIdentifier { (visitorIdentifier, error) in
 }];
 ```
 
-<Variant platform="android" api="send-queued-hits" repeat="10"/>
+<Variant platform="android" api="register-extension" repeat="8"/>
 
 #### Java
 
 **Syntax**
 
 ```java
-public static void sendQueuedHits()
+public  static void registerExtension() throws InvalidInitException
```

Comment:
Can you double check on this, Analytics probably does not throw this exception

Reply:
You are right, there is no exception is thrown.